e2e/testdata.go

```diff
@@ -61,3 +62,47 @@ func GetSamplePod(name, pvcName string) *k8sv1.Pod {
 	}
 	return pod
 }
+
+func GetSampleVolumeSnapshot(name string) *snapapi.VolumeSnapshot {
+	vscname := "odf-lvm-vg1"
+	vs := &snapapi.VolumeSnapshot{
+		ObjectMeta: metav1.ObjectMeta{
+			// name = name of pvc + snapshot
+			Name:      name + "snapshot",
+			Namespace: TestNamespace,
+		},
+		Spec: snapapi.VolumeSnapshotSpec{
+			VolumeSnapshotClassName: &vscname,
+			Source: snapapi.VolumeSnapshotSource{
+				PersistentVolumeClaimName: &name,
+			},
+		},
+	}
+	return vs
+}
+
+func GetSampleClone(quantity, name string, volumemode k8sv1.PersistentVolumeMode) *k8sv1.PersistentVolumeClaim {
+	storageClassName := "odf-lvm-vg1"
```

Comment:
Please pass this as an argument. Then if we change the storageclass name, it only has to be changed in one place. Comment holds for all instances of this usage.

Reply:
Done